src/main/scala/akka/persistence/cassandra/query/EventsByPersistenceIdPublisher.scala

```diff
@@ -0,0 +1,56 @@
+package akka.persistence.cassandra.query
+
+import scala.concurrent.duration.FiniteDuration
+
+import akka.actor.Props
+import akka.persistence.PersistentRepr
+import akka.persistence.query.EventEnvelope
+import com.datastax.driver.core.{PreparedStatement, Session}
+
+private[query] object EventsByPersistenceIdPublisher {
+  def props(
+      persistenceId: String, fromSeqNr: Long, toSeqNr: Long, refreshInterval: Option[FiniteDuration],
+      maxBufSize: Int, targetPartitionSize: Int, selectEventsByPersistenceId: PreparedStatement,
+      selectInUse: PreparedStatement, selectDeletedTo: PreparedStatement, session: Session,
```

Comment:
perhaps it would be worth grouping all the prepared statement into one class, to reduce the number of parameters to pass around?